tests/test_vtk.py

```diff
@@ -98,6 +99,28 @@ def test_pluto_two_units_override(vtk_file_with_units):
     assert_allclose_units(ds.mass_unit, expect_mass)
 
 
+def test_missing_inifile(vtk_file_with_units, tmp_path):
+    file = vtk_file_with_units
+    if file.get("require_inifile"):
+        tmpdir = tmp_path / "missing_inifile"
+        shutil.copytree(
+            file["path"].parent, tmpdir, ignore=shutil.ignore_patterns("*.ini")
+        )
+        with pytest.warns(
+            UserWarning, match=r"The inifile is missing for unit definitions. *"
+        ):
+            yt.load(list(tmpdir.glob("*.vtk"))[0])
+
+
+def test_incorrect_inifile(vtk_file_with_units):
+    file = vtk_file_with_units
+    if file.get("require_inifile"):
+        inifile = file["path"].parent / "incorrect_inifile"
+        if inifile.exists:
+            with pytest.warns(UserWarning, match=r"Cannot get the value of *"):
+                yt.load(file["path"], inifile=inifile)
```

Comment:
I'd much rather have these tests not use fixtures and parametrization since they really are meant to run on a single test case

Reply:
Okay, I'll explicitly pass the only path of sample data.